portfolio/src/main/webapp/script.js

```diff
@@ -86,23 +91,58 @@ function init() {
     }
 }
 
-// Initialize and add the map
+/**Function that is called when webpage is loaded. Created function because body onload doesn't support calling 3 functions in html */
```

Comment:
nit: Space in between `/**` and `Function`

Reply:
Fixed!